### ISSUE TITLE
Hide the side bar by default

### ISSUE THREAD

**Issue by giuseb** [giuseb/histologyguide#17]
in most cases, especially when using small devices (e.g., mobile phones), it is preferable to have as much screen real estate as possible.

**Comment by Brelje** [comment]
The analytics show that many people actually view the website on their phones.  This is surprising because it has no responsive rules for its layout.  The slides are responsive because they automatically utilize the entire browser window.  However, the sidebar has a fixed width (which would be difficult to change).  Perhaps automatically hiding the navigator, scale bar, magnification and view size in the toolbar, and view drop-down on small screens.

**Comment by giuseb** [comment]
Right, younger folks increasingly use their mobile phones for everything, which makes some type of responsive adaptation of the slide viewer a fairly high priority.

Incidentally, when we are in class, I always advise my students to hide the sidebar when first opening a slide. I want them to explore instead of automatically looking for the "right answer". Indeed, in [hguide.it](https://hguide.it) I automatically append "&f=1" to all newly saved URLs.

**Comment by giuseb** [comment]
I see that now the sidebar is hidden by default for narrow screens. Closing.
